projects/components/src/notification/notification.component.scss

```diff
@@ -35,5 +35,5 @@
 }
 
 .info {
-  color: $blue-3;
+  color: white;
```

Comment:
We need to invert this as well.

Reply:
@adisreyaj this was intentional. We don't have any usages of info toast anywhere.